Monika After Story/game/script-ch30.rpy

```diff
@@ -93,6 +93,10 @@ init -1 python in mas_globals:
     this_ev = None
     # the current topic, but as event object. may be None.
 
+    _this_ev_ctx = None
```

Comment:
Given this shouldn't be accessed directly, it may be cleaner to keep this defined statically and privately on MASEventContext to avoid it from being accessed, and also keep it self contained

Reply:
moved it to the class